services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
+    return list(Ticket.objects
+                .filter(movie_session_id=movie_session_id)
+                .values("row", "seat")
+                )
```

Comment:
In case there are no arguments on the same line with the closing parenthesis, your opening parenthesis should not have any arguments on the same line as well. Please, resolve this issue